sourcegraph/sourcegraph.proto

```diff
@@ -59,6 +59,58 @@ message ListResponse {
 	bool has_more = 1 [(gogoproto.moretags) = "url:\",omitempty\""];
 }
 
+// Discussion stores information about a discussion
```

Comment:
Minor, inconsistent that it's missing a period at the end of this sentence here (most other places have a period at the end).